assets/api-resource-tables/accounts/CreateFundingSource.tsx

```diff
@@ -0,0 +1,66 @@
+import { type TableContents } from "../../../app/components/base/material-ui/Table";
+import HalLink from "../HalLink";
+import { Link } from "../../../app/components/base/Typography";
+
+export default {
+  headers: ["Property", "Required", "Type", "Description"],
+  rows: [
+    [
+      [
+        "_links",
+        "required",
+        "object",
+        <span>
+          <Link href="/api-reference#links" text="HAL-JSON" /> links that
+          represent resource relationships required for creating a funding
+          source for an Account.
+        </span>,
+      ],
+      {
+        title: "Links Object",
+        headers: ["Link", "Description"],
+        rows: [
+          [
+            [
+              "treasury-account",
+              <span>
+                References the Treasury Account resource in relation to which
+                the funding source is created.
+                <Link
+                  href="https://www.dwolla.com/dwolla-connect/#cta-form-1"
+                  text=" Contact us "
+                />
+                to learn how to create a Treasury Account.
+              </span>,
+            ],
+            HalLink("treasury-account"),
+          ],
+        ],
+      },
+    ],
+    [
+      "accountNumber",
+      "required",
```

Comment:
Can we change the required value from required to yes or no?
```suggestion
      "yes",
```